tofu/geom/_core.py

```diff
@@ -6368,6 +6358,8 @@ def funcbis(*args, **kwdargs):
                     indpts[0], np.diff(indpts), pts.shape[1] - indpts[-1]
                 ]
                 vect = np.repeat(self.u, nbrep, axis=1)
+            if fill_value is None:
```

Comment:
Avoid nans, they are not handled by np.add.reduceat() => we set fill_value to 0 instead in this case (if not forced by the user)